### ISSUE TITLE
Update Drupal core to 7.56

### ISSUE THREAD

**Issue by jwfuller** [CuBoulder/express#1176]
## Context

Need to keep up with Drupal core releases.

- [x] Update and harden Drupal core by following instructions in https://github.com/CuBoulder/drupal-7.x
- [x] Update the [MINIMUM_CORE constant in express.profile](https://github.com/CuBoulder/express/blob/dev/express.profile#L8) 
- [x] Update [Travis Configuration](https://github.com/CuBoulder/express/blob/dev/.travis.yml#L56) so tests start running using this version of core.

The will [trigger a status error on any instance including locals](https://github.com/CuBoulder/express/blob/dev/modules/custom/express_version/express_version.install#L25), but we still need https://github.com/CuBoulder/express/issues/1258 before failing to adhere to MINIMUM_CORE will cause testing failures.

Changes include:

```
Drupal 7.56, 2017-06-21
-----------------------
- Fixed security issues (access bypass). See SA-CORE-2017-003.

Drupal 7.55, 2017-06-07
-----------------------
- Fixed incompatibility with PHP versions 7.0.19 and 7.1.5 due to duplicate
  DATE_RFC7231 definition.
- Made Drupal core pass all automated tests on PHP 7.1.
- Allowed services such as Let's Encrypt to work with Drupal on Apache, by
  making Drupal's .htaccess file allow access to the .well-known directory
  defined by RFC 5785.
- Made new Drupal sites work correctly on Apache 2.4 when the mod_access_compat
  Apache module is disabled.
- Fixed Drupal's URL-generating functions to always encode '[' and ']' so that
  the URLs will pass HTML5 validation.
- Various additional bug fixes.
- Various API documentation improvements.
- Additional automated test coverage.
```
